apps/generator/lib/generator.js

```
@@ -867,34 +891,165 @@ class Generator {
-        if (!valid) return log.debug(logMessage.conditionalFilesMatched(relativeSourceFile));
+  /**
+ * Validates the parameter value based on the provided validation schema.
+ *
+ * @private
+ * @param {Object} validation The validation schema to use.
+ * @param {any} parameterValue The value to validate.
+ * @param {String} matchedConditionPath The matched condition path.
+ * @return {Promise<boolean>} A promise that resolves to `true` if validation fails, `false` otherwise.
+ */
+  async validateParameterValue(validation, parameterValue, matchedConditionPath, relativeSourceDirectory,relativeTargetFile) {
+    if (validation.hasOwnProperty('not')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)